.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
+*.log
```

Comment:
seems you're missing the configuration file for your logger

you need to create a file `log4j.xml`

Reply:
I have such file